scripts/build

```diff
@@ -148,6 +155,8 @@ if [[ ! -d $STEP_DIR ]]; then
     $MAKE install-gcc install-target-libgcc || exit 34
   ) || err $?
 
+  export CXXFLAGS="$ORIGINAL_CXXFLAGS"
```

Comment:
Restores the original CXXFLAGS after adding Darwin-clang specific flags. They need to be changed back so they don't conflict with the cross-compile of musl